src/components/HyperlinkingWrapper.js

```diff
@@ -169,23 +167,21 @@ export default class HyperlinkingWrapper extends Component {
 
     getComputedWidth() {
         const { current } = this.state;
-        const { offsetWidth } = this.quill.root.parentElement;
         const defaultWidth = (current === HYPERLINKING_STATE.INITIAL) ? ICON_TOOLTIP_WIDTH : INPUT_TOOLTIP_WIDTH;
-        // tooltip should not be wider than width of the editor minus offset
-        const maxWidth = offsetWidth - 2 * MIN_OFFSET;
+        // tooltip should not be wider than width of the viewport
+        const maxWidth = window.innerWidth - 2 * MIN_OFFSET;
 
         return Math.min(defaultWidth, maxWidth);
     }
 
     getComputedTop(bottom) {
-        const { offsetTop } = this.quill.root.parentElement;
+        const { postEditorWrapper } = this.props;
 
-        return bottom + NOTCH_COMPENSATION + offsetTop;
+        return bottom + NOTCH_COMPENSATION + postEditorWrapper.getBoundingClientRect().top + window.scrollY;
```

Comment:
can you assign `postEditorWrapper.getBoundingClientRect().top + window.scrollY;` to `offsetTop` const?